src/test/java/com/aleksey5678/dividing/LongDivisionTest.java

```diff
@@ -0,0 +1,35 @@
+package com.aleksey5678.dividing;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+class LongDivisionTest {
+    private final LongDivision longDivision = new LongDivision();
+    private final static String expectedResultOfDividing = "_100│3\n" +
```

Comment:
please follow ordering of class members and divide them with empty line
static members
non-static members
constructors
methods

Reply:
Done